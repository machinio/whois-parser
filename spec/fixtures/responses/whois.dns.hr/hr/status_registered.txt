% IANA WHOIS server
% for more information on IANA, visit http://www.iana.org
% This query returned 1 object

refer:        whois.dns.hr

domain:       HR

organisation: CARNet - Croatian Academic and Research Network
address:      Josipa Marohnica 5
address:      Zagreb  10000
address:      Croatia

contact:      administrative
name:         CARNet DNS-Admin
organisation: CARNet - Croatian Academic and Research Network
address:      Josipa Marohnica 5
address:      Zagreb  10000
address:      Croatia
phone:        +385 1 66 61 686
fax-no:       +385 1 66 61 615
e-mail:       dns-admin@carnet.hr

contact:      technical
name:         CARNet Hostmaster
organisation: CARNet - Croatian Academic and Research Network
address:      Josipa Marohnica 5
address:      Zagreb  10000
address:      Croatia
phone:        +385 1 66 61 616
fax-no:       +385 1 66 61 615
e-mail:       hostmaster@carnet.hr

nserver:      HR-NS-1.CARNET.HR 161.53.160.100 2001:b68:ff:1:0:0:0:100
nserver:      N.DNS.HR 194.146.106.142 2001:67c:1010:36:0:0:0:53
nserver:      PCH.CARNET.HR 2001:500:14:6090:ad:0:0:1 204.61.216.90
ds-rdata:     63025 8 1 D2E75CC74208F81B1EE50EFF524259BDA1F72AC1
ds-rdata:     63025 8 2 1493B6F67A78F844E72953B29B7FEE9F6301F41E998BED94427C79A5AB60BAE6

whois:        whois.dns.hr

status:       ACTIVE
remarks:      Registration information: http://www.dns.hr

created:      1993-02-27
changed:      2020-11-04
source:       IANA

# whois.dns.hr

%
% From May 25, 2018, it  is mandatory  to apply  the  provisions  of the
% General Data Protection Regulation (EU) 2016/679. The basic purpose of
% the regulation is greater control for  individuals over their personal
% information.
%
% In  accordance  with  the  provisions  of  the General Data Protection
% Regulation (EU) 2016/679,  we inform you that the personal data of the
% domain holder and  the contact persons under  the domain holder are no
% longer publicly available through the domain search engine.
%
% If you believe that there is a legitimate reason for which you need to
% be provided with the data about the domain  holder please fill  in the
% Request form for personal data disclosure :
% https://domene.hr/portal/files/Request_personal_data_disclosure.pdf
%

Domain Name: google.hr
Registrar WHOIS Server: whois.dns.hr
Registrar URL: http://whois.dns.hr
Updated Date: 2012-08-28T22:00:00Z
Creation Date: 2011-09-20T22:00:00Z
Registrar Registration Expiration Date: 2021-09-20T22:00:00Z
Registrant Name: Google Hrvatska d.o.o.
Registrant Street: Ulica Petra HektoroviÄ‡a 2
Registrant City: Zagreb
Registrant State/Province: HR
Registrant Postal Code: 10000
Name Server: ns1.google.com
Name Server: ns2.google.com