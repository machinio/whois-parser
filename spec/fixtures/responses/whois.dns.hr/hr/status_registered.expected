#disclaimer
  %s == "From May 25, 2018, it  is mandatory  to apply  the  provisions  of the General Data Protection Regulation (EU) 2016/679. The basic purpose of the regulation is greater control for  individuals over their personal information. In  accordance  with  the  provisions  of  the General Data Protection Regulation (EU) 2016/679,  we inform you that the personal data of the domain holder and  the contact persons under  the domain holder are no longer publicly available through the domain search engine. If you believe that there is a legitimate reason for which you need to be provided with the data about the domain  holder please fill  in the Request form for personal data disclosure : https://domene.hr/portal/files/Request_personal_data_disclosure.pdf"


#domain
  %s == "google.hr"

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2011-09-20T22:00:00Z}

#updated_on
  %s %CLASS{time}
  %s %TIME{2012-08-28T22:00:00Z}

#expires_on
  %s %CLASS{time}
  %s %TIME{2021-09-20T22:00:00Z}


#registrar
  %s %ERROR{AttributeNotSupported}

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "Google Hrvatska d.o.o."
  %s[0].organization  == nil
  %s[0].address       == "Zagreb, Ulica Petra HektoroviÄ‡a 2, 10000"
  %s[0].city          == "Zagreb"
  %s[0].zip           == "10000"
  %s[0].state         == "HR"
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil

#admin_contacts
  %s %ERROR{AttributeNotSupported}

#technical_contacts
  %s %ERROR{AttributeNotSupported}


#nameservers
  %s %ERROR{AttributeNotSupported}
